src/WindowsUtils/include/WindowsUtils/CreateProcess.h

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef WINDOWS_UTILS_CREATE_PROCESS_H_
+#define WINDOWS_UTILS_CREATE_PROCESS_H_
+
+#include <OrbitBase/Result.h>
+#include <WindowsUtils/SafeHandle.h>
+
+#include <filesystem>
+#include <string>
+
+namespace orbit_windows_utils {
+
+// ProcessInfo is the result from a call to our CreateProcess function. The SafeHandle objects will
+// call "CloseHandle" on destruction for both the thread and process handles returned by the
+// internal call to the Windows CreateProcess function.
+struct ProcessInfo {
```

Comment:
Well the namespace is there to avoid clashes. If that's really a big problem I can come back and change it, but I feel it's pretty constraining to not allow the same class name in different namespaces. In this case it introduces a repetition, i.e. orbit_*windows*_utils::*Windows*ProcessInfo. In other cases it could force us to not choose the best name for a class. If we use both classes in the same translatiion unit, then we explicitly use the namespace in the declaration.